prefect_kubernetes/jobs.py

```diff
@@ -0,0 +1,205 @@
+"""Module to define tasks for interacting with Kubernetes jobs."""
+
+from typing import Dict, Optional
+
+from kubernetes import client
+from kubernetes.client.models import V1Job, V1JobList, V1Status
+from prefect import task
+
+from prefect_kubernetes.credentials import KubernetesCredentials
+
+
+@task
+async def create_namespaced_job(
+    body: Dict,
+    kubernetes_credentials: KubernetesCredentials,
+    namespace: Optional[str] = "default",
+    kube_kwargs: Optional[Dict] = None,
+) -> V1Job:
+    """Task for creating a namespaced Kubernetes job.
+
+    Args:
+        body: A dictionary representing a Kubernetes V1Job specification.
+        namespace: The Kubernetes namespace to create this job in.
+        kubernetes_credentials: KubernetesCredentials block
+            holding authentication needed to generate the required API client.
+        kube_kwargs: Optional extra keyword arguments to pass to the
```

Comment:
Maybe this can be `create_kwargs`

Reply:
Also, I think it's nicer on the docs if this is `**kube_kwargs`. Repeat for the other docstrings below.